tap_adroll/streams.py

```diff
@@ -70,12 +70,20 @@ class AdReports(Stream):
 
     def generate_daily_date_windows(self):
         bookmark = singer.bookmarks.get_bookmark(self.state, self.stream_name, self.replication_keys[0])
-        report_date = utils.strptime_to_utc(bookmark or self.config['start_date'])
+        lookback_window = datetime.timedelta(days=int(self.config.get('lookback_window') or 7))
```

Comment:
The `get()` function supports a default value, can you use that here instead of the `or`?